src/main/java/com/termos/scooterrental/SpringBootConsoleApplication.java

```diff
@@ -0,0 +1,196 @@
+package com.termos.scooterrental;
+
+import com.termos.scooterrental.model.Rent;
+import com.termos.scooterrental.model.Scooter;
+import com.termos.scooterrental.repository.RentRepository;
+import com.termos.scooterrental.time.TimeUtils;
+import com.termos.scooterrental.model.ScooterDock;
+import com.termos.scooterrental.model.UserAccount;
+import com.termos.scooterrental.repository.ScooterDockRepository;
+import com.termos.scooterrental.repository.ScooterRepository;
+import com.termos.scooterrental.repository.UserAccountRepository;
+import lombok.AllArgsConstructor;
+import lombok.Lombok;
+import org.springframework.boot.Banner;
+import org.springframework.boot.CommandLineRunner;
+import org.springframework.boot.SpringApplication;
+import org.springframework.boot.autoconfigure.SpringBootApplication;
+
+import java.math.BigDecimal;
+import java.math.BigInteger;
+import java.sql.Timestamp;
+import java.time.LocalDate;
+import java.time.LocalDateTime;
+import java.util.List;
+import java.util.Scanner;
+
+import static java.lang.System.exit;
+import static java.lang.System.setOut;
+
+@SpringBootApplication
+@AllArgsConstructor
+public class SpringBootConsoleApplication implements CommandLineRunner {
+    ScooterDockRepository scooterDockRepository;
```

Comment:
Pola klasy powinny mieć jak najmniejszą widoczność. Czyli jeśli nie jest to konieczne powinny być prywatne.